ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1314,3 +1314,9 @@ Display = Display
 date.time.title = Date & time 
 admin.display = Display
 admin.display.datetime = Date & time 
+edit.name = Edit name
+display.language.accordion.title = Enabled languages will be available to users as an option when they select their primary language. We recommend picking 1-3 supported languages and disabling the rest. Of the enabled supported languages, there should be 0 missing translations.
+display.translation.description = Please note that any changes or additions you make here to the translations are not currently being saved or tracked anywhere else.
+edit.language.description = Ideally any new languages added would incorporated into the CHT core so that they can be made available to other partners. Please reach out if you have questions about adding a new language.
+translation.add.key = Add translation key
+edit.language.code.title = The recommended country codes follow the ISO standard, which can be referenced here.
```

Comment:
The word "here" is supposed to link to wikipedia. This is hard to do with translations because you can't include markup.

Change the template to be:

```
<span class="help-block">
  <span translate>Language code help</span>
  <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Current_codes" translate>edit.language.code.help.link</a>
</span>
```

Then add the translation key: `edit.language.code.help.link = ISO standard`
Then edit the translation key: `Language\ code\ help = The 2 or 3 digit code for the language following the`

I think that'll work...